keyboards/planck/keymaps/am/config.h

```diff
@@ -0,0 +1,45 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```

Comment:
Could you remove this.  It's not needed and a hold over from the old makefile system.